beetsplug/info.py

```diff
@@ -119,6 +119,24 @@ def print_data(data, item=None, fmt=None):
         ui.print_(lineformat.format(field, value))
 
 
+def print_data_keys(data, item=None):
+
+    path = displayable_path(item.path) if item else None
+    formatted = []
+    for key, value in data.iteritems():
+        formatted.append(key)
+
+    if len(formatted) == 0:
+        return
+
+    line_format = u'{0}{{0}}'.format(u' ' * 4)
+    if path:
+        ui.print_(displayable_path(path))
```

Comment:
The problem with the `cut -d: -f1` is when the item's path is printed, like d:\music